tern/analyze/docker/dockerfile.py

```diff
@@ -265,3 +265,38 @@ def get_base_image_tag(base_instructions):
     if len(image_tag_list) == 1:
         image_tag_list.append('')
     return tuple(image_tag_list)
+
+
+def find_git_info(line, dockerfile_path):
+    '''Given a line of ADD command and the absolute path of dockerfile,
+    return the information(string format) on the git project name and sha
+    if a folder or tar containing a git repository.
+    ADD command has a general format:
+    ADD [--chown=<user>:<group>] <src> <dst>
+    '''
+    args = line.split(' ')
+    src_path = ''
+    # check if --chown exists
+    if args[1].startswith('--chown'):
+        # check if the line is valid
+        if len(args) < 4:
+            raise ValueError('Invalid ADD command line')
```

Comment:
Instead of raising a `ValueError` can you log the error? If we raise the error it will cause Tern to quit and ideally we would continue with the rest of the potential lines in the Dockerfile. Something like:

```
logger.error("Invalid ADD command line: %s", line)
```